### PR TITLE
Feature: Group Event Import

### PR DESCRIPTION
**Pull request by r-a-y**
We did this for the Commons, but we might want to consider merging this branch into 1.1.x or consider doing this in a future release.

To summarize, this PR allows for groups to import events from an uploaded ICS file or to import events from an iCalendar URL. Importing from an iCalendar URL will automatically set up a cronjob for the group to sync events every hour.

**Note:** The iCalendar URL import requires [Event Organiser's iCal Sync](https://wp-event-organiser.com/extensions/event-organiser-ical-sync/) premium extension

### CODE REVIEW

File: bp-event-organiser-groups.php [hunk 1]
```diff
@@ -617,6 +620,16 @@ public function edit_screen_callback( $group_id = null ) {
 
 			// remove our template stack
 			remove_filter( 'eventorganiser_template_stack', 'bpeo_register_template_stack' );
+
+		/*
+		 * Public group, so add dummy submit button.
+		 *
+		 * The BP Group Extension API adds a submit button by default if one doesn't
```
Comment (boonebgorges) [inline bp-event-organiser-groups.php:627]: Ha ha. Sorry about that.

**Review by boonebgorges — APPROVED** [review]
Yeah, I think we should definitely include this. Can't see a reason not to.

**Comment by r-a-y** [comment]
Just to update, this PR was made against 1.1.x branch, but I'm about to merge this into 1.2.x branch and re-tag 1.2.0, pending a quick review of other issues.

**Comment by r-a-y** [comment]
`ical-sync` branch has been merged into 1.2.x branch. I've retagged 1.2.0 [here](https://github.com/cuny-academic-commons/bp-event-organiser/releases/tag/1.2.0).

Closing this one out.

I do have a question about whether we want to expose the group admin's "Settings > Events" breadcrumb link for `openlab-theme`. Right now, it isn't shown. Should I create an issue over on the CBOX repo?

**Comment by boonebgorges** [comment]
Thanks for taking the initiative to clean up the 1.2.0 tag, @r-a-y !

Yes, if you could open a ticket on the CBOX repo, that would be great. Ideally, you could include a link to whatever part of the bp-event-organiser plugin is responsible for building the markup on default BP templates, to make it easier for me to port :)

**Pull request closed without merging**